dotcom-rendering/src/web/lib/decideLogo.ts

```diff
@@ -27,7 +28,7 @@ const shouldUseLogoForDarkBackground = (format: ArticleFormat): boolean => {
 export const decideLogo = (
 	format: ArticleFormat,
 	branding: Branding,
-): BrandingLogo => {
+): Branding['logo'] => {
```

Comment:
Mapped types here too, to reduce the amout of exports. Happy to export `BrandingLogo` explicitely